annif/swagger/annif.yaml

```diff
@@ -143,6 +143,12 @@ definitions:
         example: en
       backend:
         $ref: '#/definitions/ProjectBackend'
+      is_trained:
+        type: boolean
+        example: true
+      modification_time:
+        type: string
```

Comment:
you could add `format: date-time` here to indicate that the string is in datetime format